doc/api/cli.md

```diff
@@ -490,6 +490,40 @@ For example, to run a module with "development" resolutions:
 node -C development app.js
 ```
 
+### `--coverage-exclude-glob`
```

Comment:
```suggestion
### `--coverage-exclude`
```

seems redundant to me

Reply:
also it should probably be `--test-coverage-exclude` and `--test-coverage-include` like all other test runner related flags start with `--test`. (and perhaps `--experimental-test-coverage-exclude` & `--experimental-test-coverage-include`)

Reply:
`--test-coverage-<...>` seem good to me. I don't think they need to be `--experimental-` prefixes, as they can *only* be used when `--experimental-test-coverage` is already enabled.